### ISSUE TITLE
Global Style Page not showing different font sizing

### ISSUE THREAD

**Issue by JennyPower** [elementor/elementor#22831]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Not sure if I'm missing something.
The new Global Style Guide page shows all the headings the same size when they are all different sizes in the typography settings.
Potentially this could be because I'm using clamp and maybe that's not supported
I have my own style guide page which shows the different heading sizes just fine.

### Steps to reproduce

1) Set up Global Fonts at different sizes under Site Settings using clamp css eg. clamp(32px, 4vw, 48px)
2) Apply these fonts to headings in Typography
3) View the new Global Style Guide and see if the fonts display at different sizes.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.14
	PHP Version: 8.0.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://webtree.website/nbpr
	Home URL: https://webtree.website/nbpr
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-AU
	Timezone: Australia/Sydney
	Admin Email: jenny@webtree.com.au
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_AU
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Duplicator Pro
		Version: 4.5.11.1
		Author: Duplicator

	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com

	ManageWP - Worker
		Version: 4.9.17
		Author: GoDaddy

	Wordfence Security
		Version: 7.9.3
		Author: Wordfence

	WP-Optimize - Clean, Compress, Cache
		Version: 3.2.15
		Author: David Anderson, Ruhani Rabin, Team Updraft

	Yoast SEO
		Version: 20.9
		Author: Team Yoast

== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 
		Author: GoDaddy

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Inactive by default
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Active
	Page Transitions: Active
	Notes: Inactive
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Inactive by default

== Log ==
	
Log: showing 20 of 362023-05-30 09:45:08 [info] elementor::elementor_updater Started 
2023-05-30 09:45:08 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-30 09:45:08 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-30 09:45:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.13.4',
)]
2023-05-30 09:45:08 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.13.4',
)]
2023-06-05 11:05:45 [info] elementor-pro::elementor_pro_updater Started 
2023-06-05 11:05:45 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-05 11:05:45 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-05 11:05:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-06-05 11:05:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-06-20 13:08:34 [info] elementor::elementor_updater Started 
2023-06-20 13:08:34 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-20 13:08:34 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-20 13:08:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]
2023-06-20 13:08:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]
2023-06-20 13:57:40 [info] elementor-pro::elementor_pro_updater Started 
2023-06-20 13:57:40 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-20 13:57:40 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-20 13:57:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-20 13:57:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]

JS: showing 2 of 2JS: 2023-05-23 03:15:01 [error X 4][https://webtree.website/nbpr/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.13.1:2:63893] Cannot convert undefined or null to object 
JS: 2023-06-20 04:16:12 [error X 2][https://webtree.website/nbpr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.0:3:54034] Cannot read properties of undefined (reading 'activate') 

PHP: showing 4 of 4PHP: 2023-06-20 13:44:02 [error X 4][/home/webtreew/public_html/nbpr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in /home/webtreew/public_html/nbpr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 /home/webtreew/public_html/nbpr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/webtreew/public_html/nbpr/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /home/webtreew/public_html/nbpr/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/webtreew/public_html/nbpr/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /home/webtreew/public_html/nbpr/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /home/webtreew/public_html/nbpr/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-20 14:00:49 [warning X 7][/home/webtreew/public_html/nbpr/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-20 16:04:26 [warning X 1][/home/webtreew/public_html/nbpr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-20 16:10:13 [warning X 1][/home/webtreew/public_html/nbpr/wp-content/plugins/elementor/app/modules/import-export/module.php::734] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hello @JennyPower! 

Thank you for submitting your issue! 🙏 

🏷️ We've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding!

Best regards

**Comment by natsamson** [comment]
Exactly the same problem 6 months later...

**Comment by xetyr** [comment]
Yep, I would enjoy a fix too !

**Comment by Bobj2018** [comment]
Yeah, it would be great if this was fixed.

**Comment by unionglyph** [comment]
I am having the same issue

**Comment by pauljmartinez** [comment]
I have the same issue and it seems to be specific to the use of clamp when setting the sizes. The problem appears consistently but I've been unable to create a reliable test case. For example, some of my fonts that use clamp are working fine, while others are not working. I'd be happy to supply any debugging info if that would be helpful.

**Comment by bruceyoong** [comment]
I have the same issue, when I used the px to adjust the global font size, the view updated right away, but when I used the clamp function, it didn't work correctly.

**Comment by georgevanvorhis** [comment]
Still experiencing this issue. Wasted a lot of time trying to figure it out thinking it was a theme conflict or something </3

**Comment by baraklevy** [comment]
I just found out that Elementor deleted the spaces in the function! This is the reason it is not working.
I used: font-size: clamp(2.875rem, 2.0938rem + 2.5vw, 3.5rem);
in the editor, it looks fine, but on the front, it converted to font-size: clamp(2.875rem, 2.0938rem+2.5vw, 3.5rem); - 
the two spaces around the plus are gone. is there any solution for this????
